examples/speak/rest/text_to_speech_to_file.rs

```diff
@@ -10,10 +10,10 @@ async fn main() -> Result<(), DeepgramError> {
     let dg_client = Deepgram::new(&deepgram_api_key);
 
     let options = Options::builder()
-        .model("aura-asteria-en")
-        .encoding("linear16")
+        .model(Model::AuraAsteriaEN)
```

Comment:
Standard rust practice is to only capitalize the first letter of all-caps names.  Like `Html`, `IpAddr`, and here, `AuraAsteriaEn`.